### PR TITLE
Fix one grammar error and a broken link in PROTOCOL.md

### PR DESCRIPTION
**Pull request by vegarsti**
Hi all, thanks for this terrific project. I was reading through PROTOCOL.md and spotted these two mistakes.

### CODE REVIEW

File: PROTOCOL.md [hunk 1]
```diff
@@ -462,7 +462,7 @@ Within the checkpoint, the `add` struct may or may not contain the following col
  ```
 
  - stats: Column level statistics can be stored as a JSON string in the checkpoint. This field needs to be written when statistics are available and the table property: `delta.checkpoint.writeStatsAsJson` is set to `true` (which is the default). When this property is set to `false`, this field should be omitted from the checkpoint.
- - stats_parsed: The stats can be stored in their [original format](#Per-file Statistics). This field needs to be written when statistics are available and the table property: `delta.checkpoint.writeStatsAsStruct` is set to `true`. When this property is set to `false` (which is the default), this field should be omitted from the checkpoint.
+ - stats_parsed: The stats can be stored in their [original format](#Per-file-Statistics). This field needs to be written when statistics are available and the table property: `delta.checkpoint.writeStatsAsStruct` is set to `true`. When this property is set to `false` (which is the default), this field should be omitted from the checkpoint.
```
Comment (scottsand-db) [inline PROTOCOL.md:465]: let's keep it the way it was please.

> Use a hyphen to join two or more words to form compound adjectives that precede a noun. The purpose of joining words to form a compound adjective is to differentiate the meaning from the adjectives used separately, such as up-to-date merchandise, copper-coated wire, fire-tested material, lump-sum payment, and well-stocked cupboard.

http://www.butte.edu/departments/cas/tipsheets/punctuation/hyphen.html

**Review by scottsand-db — CHANGES_REQUESTED** [review]

File: PROTOCOL.md (hunk 1 above)
Comment (zsxwing) [reply PROTOCOL.md:465]: I think this is to fix the link. `#Per-file Statistics` is not a valid link.

Comment (vegarsti) [reply PROTOCOL.md:465]: Exactly!

Comment (vegarsti) [reply PROTOCOL.md:465]: Thanks for responding! I should have indicated more clearly that this fixed the link

Comment (scottsand-db) [reply PROTOCOL.md:465]: haha. makes sense

**Review by scottsand-db — APPROVED** [review]

**Comment by vegarsti** [comment]
Thanks! Feel free to merge this when you feel like it.

**Pull request closed without merging**